src/quo2/screens/account_selector.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.screens.account-selector
+  (:require [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.account-selector :as quo2]))
+
+(def descriptor  [{:label "Show Label?:"
+                   :key   :show-label?
+                   :type  :boolean}
+                  {:label "Transparent Background?:"
+                   :key   :transparent?
+                   :type  :boolean}
+                  {:label "Account Text"
+                   :key   :account-text
+                   :type  :text}
+                  {:label "Label Text"
+                   :key   :label-text
+                   :type  :text}])
+
+(defn cool-preview []
+  (let [state    (reagent/atom {:show-label?    true
+                                :transparent?   false
+                                :style          {:width :100%}
```

Comment:
why` :100%` is a keyword? and also it's better to do not use percents in UI

Reply:
Okay shall get rid of it